tasks/minesweeper/README.md

```diff
@@ -108,5 +110,15 @@ Recommended usage of [eslint (eslint-config-airbnb-base)](https://eslint.org/),
 - [Local storage](https://learn.javascript.ru/localstorage)
 - [How-to play in minesweeper](https://www.youtube.com/watch?v=7B85WbEiYf4)
 
-[The form for students questions](https://docs.google.com/forms/d/1rl3fHYRJLAKpYo4T5t6wRrixxMChunSX6KnEkBSd170)  
-[Please, provide the best apps from cross-check in the following form](https://docs.google.com/forms/d/1HR5K7fsIbLNCC0UNcJ_67OkOBptkEsIl4ndtzL63yvM/edit)
+FAQ
+|Question|Answer|
+|---|---|
+|When game starts?|After player clicked on empty cell using left mouse button|
+|May I use React/Angular/View, MobX, etc.?|For this task - no|
+|May I use images except color numbers?|If images are used for different numbers, flags and mines, they should differ (one image for number 1, another for number 2, etc.)|
+|How many bombs should there be by default for a field size: 10x10, 15x15, 25x25?|Additionally, users should have the ability to manually select the number of mines for each field size (from 10 to 99). The way of implementing the choice of the number of mines (numeric input, input range, etc.) is at your discretion|
+|Is it necessary to put up all the flags in right places with bombs to win the game?|Flags do not affect the outcome of the game; they are merely a supplementary tool for the player's convenience.|
+|I noticed a feature in original classic game which is not described in task description. Should I implement it?|No need to implement any other features from original minesweeper which is not noticed above.|
+|What if player mark a sell with flag before he clicking on field using left mouse click? May the mine be in this cell?|If the player placed a mine on the field before starting the game (clicked on the cell with the left mouse button) - under the flag there may be either a mine, an empty cell or a cell with a number. Placing a flag does not start the game and mines will be formed after the first click on the cell left mouse button|
```

Comment:
It's not 100% clear what is meant by this phrase:

_If the player placed a mine on the field before starting the game (clicked on the cell with the left mouse button)_

If we assume the word `flag` should go instead of `mine`, does it mean that player placed a flag by clicking on the cell with the left mouse button or he started a game by clicking on the cell with the left mouse button? 

Maybe it would be better this way:
_If the player flags a cell before starting the game (by left-clicking on the cell),..._

I'm sorry if I misunderstand your idea with mines/flags in this sentence